README.md

```diff
@@ -17,6 +17,8 @@ npx cap sync
 * [`disable()`](#disable)
 * [`enableScanner()`](#enablescanner)
 * [`disableScanner()`](#disablescanner)
+* [`startScanning()`](#startscanning)
+* [`disableScanner()`](#stopscanning)
```

Comment:
Probably forgot to docgen regenerate before pushing? should be stopScanning() here